### ISSUE TITLE
Get alias and userid of the Zabbix Group Members

### ISSUE THREAD

**Issue by AlexSamad** [BernhardLinz/zabbix-ldap-sync-bash#6]
In the code under this section

I don't see where alias is used 
and the code after it only looks for stuff in the ZABBIX_ARRAY_LDAP_GroupMember_alias array

https://www.zabbix.com/documentation/current/en/manual/api/reference/user/get

doesn't have an alias attribute ?

Not sure if this was something from before . username is a must now

**Comment by BernhardLinz** [comment]
The script is tested with Zabbix 3.0 to 5.0 and "it works". I will test 6.0 when it is released
i had have make changes for 5.4, there is a check for the Zabbix-Version. Up to 5.0.0 the "User-Type-Mode" is used, if Version is higher than 5.0.999 the "User-RoleId-Mode" is used. Check for the $s_UserMode Variable in source code.

**Comment by elshandra** [comment]
Easily fixed for 6.0 by changing "alias" to "username" on line 816,951 ie:
816: if [ "${ZABBIX_ARRAY_LDAP_GroupMember_RAW[$i]}" = "username" ]; then
951: if [ "${ZABBIX_ARRAY_AllUser_RAW[$i]}" = "username" ]; then

**Comment by BernhardLinz** [comment]
I change the the two lines to:
```
816:  if [ "${ZABBIX_ARRAY_LDAP_GroupMember_RAW[$i]}" = "alias" ] || [ "${ZABBIX_ARRAY_LDAP_GroupMember_RAW[$i]}" = "username" ]; then
951: if [ "${ZABBIX_ARRAY_AllUser_RAW[$i]}" = "alias" ] || [ "${ZABBIX_ARRAY_AllUser_RAW[$i]}" = "username" ]; then

```
this add an "or" for username or alias, it should work with all Zabbix Version and i need no extra Check for Version 6.x (at this time)
I Update the Master-Version to 1.3.2

Thanks @elshandra  for the quick solution
